backstory/src/main/java/com/google/sps/perspective/data/ContentDecisions.java

```diff
@@ -69,4 +71,104 @@ private static boolean isToxic(Map<AttributeType, Float> attributeTypesToScores)
 
     return toxicity >= .7f;
   }
+
+  /**
+   * Private helper method to check if content is considered sexually explicit.
+   * Threshold for sexually explicit content is a score greater than or equal to 60%
+   * after experimenting with Perspective API
+   *
+   * @param attributeTypesToScores a map with attribute types mapped to scores from the Perspective
+   *     API
+   * @return true, if sexually explicit score >= 60%; false, if not
+   * @throws IllegalArgumentException, if attributeTypesToScores is null or does not contain a
+   *     sexually explicit score
+   */
+  private static boolean isSexuallyExplicit(Map<AttributeType, Float> attributeTypesToScores)
+      throws IllegalArgumentException {
+    if (attributeTypesToScores == null) {
+      throw new IllegalArgumentException("Map (attributeTypesToScores) cannot be null.");
+    } else if (!attributeTypesToScores.containsKey(AttributeType.SEXUALLY_EXPLICIT)) {
+      throw new IllegalArgumentException(
+          "Map (attributeTypesToScores) does not contain a sexually explicit score");
+    }
+
+    float sexuallyExplicit = attributeTypesToScores.get(AttributeType.SEXUALLY_EXPLICIT);
+
+    return sexuallyExplicit >= .6f;
```

Comment:
Maybe listing these thresholds as constants at the top of the file would make them easier to find and change.

Making them public would then allow you to reference the constants from the tests, which could reduce test maintenance:

```
void checkDecisionForHighSexuallyExplicit() {
  final float HIGH_SEXUALLY_EXPLICIT = ContentDecisions.MAX_APPROPRIATE_SEXUALLY_EXPLICIT_SCORE + 0.1f;
 ...
}
```

Reply:
This is a great suggestion. I'll implement this.

Reply:
Done